pkg/addons/data/StorageClass.storage.k8s.io/azure.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   name: azure
 parameters:
   kind: managed
-  location: eastus
+  location: "******* Generated *******"
```

Comment:
Nit: I think all of our generated fields use 3 stars and all caps.  Nothing depends on it but might as well keep them the same. 

Other than that, lgtm.  Tag when updated.